doc/PARALLAX_IMAGE.md

```diff
@@ -14,6 +14,7 @@ Prop | Description | Type | Default
 `parallaxFactor` | Speed of the parallax effect. Be aware that the bigger the value, the more image will appear "zoomed in". | Number | `0.3`
 `showSpinner` | Whether to display a spinner while image is loading or not | Boolean | `true`
 `spinnerColor` | Color of the spinner | String | 'rgba(0, 0, 0, 0.4)'
+`animatedImage` | Custom animated image component | Function Object | `Animated.Image`
```

Comment:
IMO `AnimatedImageComponent` would be more descriptive.

Reply:
Agree, done